packages/liquidator/index.js

```diff
@@ -89,9 +89,11 @@ async function run({
       ).length == 0
     )
       throw new Error(
-        `Contract version specified or inferred is not supported by this bot. Provided/inferred config: ${JSON.stringify(
```

Comment:
change made here helps with debugging as it prints out the whole `detectedContract`